Lib/_collections_abc.py

```diff
@@ -518,8 +518,9 @@ def _type_repr(obj):
     Copied from :mod:`typing` since collections.abc
     shouldn't depend on that module.
     """
-    if isinstance(obj, GenericAlias):
-        return repr(obj)
+    # When changing this function, don't forget about
+    # `typing._type_repr` function that does the same thing
+    # and must be consistent with this one.
```

Comment:
I don't think we need _this_ comment, because it's mentioned in the docstring directly above that it's copied from `typing`

```suggestion
```

Reply:
Yeah, but I still went for it. Because for me "copied" does not apply "must be kept in sync".

It won't hurt, in my opinion :)